src/java/com/zimbra/qa/selenium/projects/admin/tests/domains/ConfigureGAL.java

```diff
@@ -210,6 +210,7 @@ public void ConfigureGAL_03() throws HarnessException {
 		gItem.setNewGALMode(GALMode.Both);
 		gItem.setDataSourceName(intDataSrc);
 		gItem.setPollingIntervalDays("1");
+		gItem.setBindPassword("z1mbr4@123");
```

Comment:
Can we add/take password from config?

Reply:
This is same as what we giving while creating domain in same test.
![image](https://user-images.githubusercontent.com/27402349/99995859-ff0e8d00-2de0-11eb-8cc5-a019700d9f49.png)